libraries/AP_ESC_Telem/AP_ESC_Telem_Backend.h

```diff
@@ -42,6 +46,10 @@ class AP_ESC_Telem_Backend {
         USAGE       = 1 << 5,
         TEMPERATURE_EXTERNAL = 1 << 6,
         MOTOR_TEMPERATURE_EXTERNAL  = 1 << 7,
+#ifdef HAL_WITH_BIDIR_DSHOT
```

Comment:
Could we get a separate define for this, V2 functionality, please?
```suggestion
#ifdef HAL_WITH_BIDIR_DSHOT_V2
```

It can be defined as
```
#define HAL_WITH_BIDIR_DSHOT_V2 HAL_WITH_BIDIR_DSHOT
```
... but it would be nice to have the option.

Reply:
I will address this next time I update, following checking the IOMCU functionality and syncing everyone's thoughts on where exactly logging should go.